.github/workflows/kubernetes_test.yaml

```diff
@@ -32,9 +33,9 @@ jobs:
       run:
         shell: bash -l {0}
     steps:
-      - name: "Set QHUB_IMAGE_TAG=main"
+      - name: "Set QHUB_IMAGE_TAG=develop"
```

Comment:
This hardcodes for `develop` and drops `main` altogether - the idea is that it works for both not one or the other. 
Could leverage GH actions context to get the branch/ref name and set here